database/factories/UserFactory.php

```diff
@@ -41,11 +40,3 @@
         },
     ];
 });
-
-$factory->state(User::class, 'with-team', function (\Faker\Generator $faker) {
```

Comment:
Why did you removed this `state`? It's not useful anymore?

Reply:
No, it is not useful since it creates a team but does not add the user as participant, only as owner, so it would be more confusing.

Reply:
Ok so should't we add a `team` in the default user _factory_ just to make it more complete?

Reply:
I don't think so, because it is not useful in tests. Let's say that in our logic we want that a user that owns the team is also considered a participant. Using factories, we cannot do that since only the owner would be specified. So I opted out to make it by hand in tests. In fact, it is still confusing, but is concentrated only in tests.

Reply:
I agree :+1: